modules/op-release-notes-1-15.adoc

```diff
@@ -0,0 +1,430 @@
++
+.Example of a `TriggerTemplate` CR
+[source,yaml]
+----
+apiVersion: triggers.tekton.dev/v1beta1
+kind: TriggerTemplate
+metadata:
+  name: create-configmap-template
+spec:
+  params:
+    - name: action
+  resourcetemplates:
+    - apiVersion: v1
+      kind: ConfigMap
+      metadata:
+        generateName: sample-
+      data:
+        field: "Action is : $(tt.params.action)"
+----
+
+* With this update, you can define the `ServiceType` in an `EventListener` CR as `NodePort` and define the port number for the event listener, as shown in the following example:
++
+.Example of an `EventListener` CR defining a port number
+[source,yaml]
+----
+apiVersion: triggers.tekton.dev/v1beta1
+kind: EventListener
+metadata:
+  name: simple-eventlistener
+spec:
+  serviceAccountName:  simple-tekton-robot
+  triggers:
+    - name: simple-trigger
+      bindings:
+      - ref: simple-binding
+      template:
+        ref: simple-template
+  resources:
+    kubernetesResource:
+      serviceType: NodePort
+      servicePort: 38080
+----
+
+* With this update, if you use a `serviceType` value of `LoadBalancer` in an `EventListener` CR, you can optionally specify a load balancer class in the `serviceLoadBalancerClass` field. If your cluster provides multiple load balancer controllers, you can use the load balancer class to select one of these controller. For more information about setting a load balancer class, see the link:https://kubernetes.io/docs/concepts/services-networking/service/#load-balancer-class[Kubernetes documentation].
++
+.Example of specifying a LoadBalancerClass setting
+[source,yaml]
+----
+apiVersion: triggers.tekton.dev/v1beta1
+kind: EventListener
+metadata:
+  name: listener-loadbalancerclass
+spec:
+  serviceAccountName: tekton-triggers-example-sa
+  triggers:
+    - name: foo-trig
+      bindings:
+        - ref: pipeline-binding
+        - ref: message-binding
+      template:
+        ref: pipeline-template
+  resources:
+    kubernetesResource:
+      serviceType: LoadBalancer
+      serviceLoadBalancerClass: private
+----
+
+[id="manual-approval-new-features-1-15_{context}"]
+=== Manual approval
+
+With this update, {pipelines-shortname} includes the new Manual Approval Gate functionality.
+
+Manual Approval Gate is a Custom Resource Definition (CRD) controller. You can use this controller to add manual approval points in the pipeline so that the pipeline waits at that point and waits for a manual approval before continuing execution. To use this feature, specify an `ApprovalTask` in the pipeline, in a similar way to specifying a `Task`. Users can provide the approval using the web console or using the `tkn` or `opc` command line utility.
```

Comment:
```suggestion
Manual Approval Gate is a Custom Resource Definition (CRD) controller. You can use this controller to add manual approval points in the pipeline so that the pipeline waits at that point and waits for a manual approval before continuing execution. To use this feature, specify an `ApprovalTask` in the pipeline, in a similar way to specifying a `Task`. Users can provide the approval using the web console or using the  `opc` command line utility.
```